expression/expr_to_pb.go

```diff
@@ -182,6 +182,10 @@ func (pc PbConverter) columnToPBExpr(column *Column) *tipb.Expr {
 		return nil
 	}
 	switch column.GetType().Tp {
+	case mysql.TypeBit:
+		if !IsPushDownEnabled("bit", kv.TiKV) {
```

Comment:
how about TiFlash?

Reply:
No，The bit column won't be push down to tiflash due to following codes:

https://github.com/pingcap/tidb/blob/493eb45c75a8f9687d5bf9185e5e167aadc81012/expression/expression.go#L1275-L1285